.changeset/twenty-cameras-take.md

```diff
@@ -1,5 +1,5 @@
 ---
-"@fluid-experimental/presence": minor
+"@fluidframework/presence": minor
```

Comment:
@tylerbutler, under a rename, should this have both package names?

Reply:
No, it should only have the new name. Otherwise, the changelog generation tools (we use ecosystem tools for this) complains. We should note the rename in the changeset.